### ISSUE TITLE
Improvement: Use POST to avoid recording domains in server log

### ISSUE THREAD

**Issue by youknow16** [checkmyhttps/checkmyhttps#48]
Right now, the script contacts the server using a GET or HEAD method. The problem with this is that every domain checked will be recorded in the server logs.
It would be best if the script contacts server using  a POST method so that the checks don't end up in the server logs.

**Comment by checkmyhttps** [comment]
Yes, you are right. We write this change request in our to-do list. Thx

**Comment by Rubber-Duckie** [comment]
Guys I see you haven’t improved this for years. any reason.

**Comment by checkmyhttps** [comment]
See last commits. It's in progress
